### PR TITLE
Fix: Limit the number of display assets and add stats icon to the footer

### LINKED ISSUE #2484: Bug: Assets and labels on OI stats are overlapping
### Subject of the issue

With the increasing number of markets the OI stats chart is becoming too crowded and causing layout issues. 

Screenshot 2023-06-13 at 21 57 42

### Possible Solution 

Reduce icon size and make the labels vertically aligned

### PR DESCRIPTION
**Pull request by LeifuChen**
## Description
With the increasing number of markets, the OI stats chart is becoming too crowded, causing layout issues.
截屏2023-07-25 00 33 04

In this PR, the following changes have been made.
1. The number of asset icons in the Open Interests chart is adjusted according to the screen width. Then we combined all the hidden markets into `Others`.

| Device  | No. Assets |
|---------|------------|
| mobile  |          4 |
| tablet  |         14 |
| desktop |         24 |

2. The chart is re-rendered after resizing the screen width.
3. Add the stats icon to the page footer
截屏2023-07-25 22 39 07

## Related issue
Closes #2484 

#### Desktop: 25 assets
截屏2023-07-25 20 48 43

#### Tablet: 15 assets
截屏2023-07-25 20 46 29

#### Mobile: 5 assets
截屏2023-07-25 20 48 01

### CODE REVIEW

**Review by avclarke — COMMENTED** [review]
Changes look good but was wondering if we should maybe combine the hidden markets to `other`? @LeifuChen

**Comment by LeifuChen** [comment]
> Changes look good but was wondering if we should maybe combine the hidden markets to `other`? @LeifuChen

Good idea. Let's do it.

**Review by Marko-roy — DISMISSED** [review]
✅

**Review by platschi — CHANGES_REQUESTED** [review]
Not sure why, but on the latest deployment the OI chart does not display on first load. Only after going to the app, connecting to the wallet and returning to the Stats page, the OI data appears.

You can verify the issue in an incognito window.

Screenshot 2023-07-25 at 09 55 49

**Comment by LeifuChen** [comment]
> Not sure why, but on the latest deployment the OI chart does not display on first load. Only after going to the app, connecting to the wallet and returning to the Stats page, the OI data appears.
> 
> You can verify the issue in an incognito window.
> 
> Screenshot 2023-07-25 at 09 55 49

It looks like a network issue. Have you tried this again?

**Comment by LeifuChen** [comment]
> Not sure why, but on the latest deployment the OI chart does not display on first load. Only after going to the app, connecting to the wallet and returning to the Stats page, the OI data appears.
> 
> You can verify the issue in an incognito window.
> 
> Screenshot 2023-07-25 at 09 55 49

Thanks for the report.  I suspect that the `fetchMarkets` function may not have been called if the user didn't visit the market or dashboard page. It should be fixed in the latest deployment.

**Review by avclarke — APPROVED** [review]
Nice work, looks good!

File: packages/app/src/sections/shared/Layout/AppLayout/Footer.tsx [hunk 1]
```diff
@@ -11,6 +12,7 @@
 
 import GitHashID from './GitHashID'
 import OperationStatus from './OperationStatus'
+import { Stats } from 'sections/stats'
```
[the remark this answers is not in the document]
Comment (avclarke) [reply packages/app/src/sections/shared/Layout/AppLayout/Footer.tsx:15]: @LeifuChen just this warning

[the remark this answers is not in the document]
Comment (platschi) [reply packages/app/src/sections/shared/Layout/AppLayout/Footer.tsx:15]: Fixed

**Review by platschi — APPROVED** [review]

**Pull request merged**